include/gz/physics/config.hh.in

```diff
@@ -27,6 +27,7 @@
 
 #define GZ_PHYSICS_VERSION "${PROJECT_VERSION}"
 #define GZ_PHYSICS_VERSION_FULL "${PROJECT_VERSION_FULL}"
+#define GZ_PHYSICS_VERSION_NAMESPACE v${PROJECT_VERSION_MAJOR}
```

Comment:
per https://github.com/gazebosim/gz-cmake/issues/412, this breaks ABI

it looks like this PR was merged even with a broken ABI build